test/parallel/test-repl-eval.js

```
@@ -0,0 +1,22 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const repl = require('repl');
+
+{
+  const options = {
+    eval: common.mustCall((cmd, context) => {
+      assert.strictEqual(cmd, 'foo\n');
+      assert.deepStrictEqual(context, {animal: 'Sterrance'});
+    })
+  };
+
+  const r = repl.start(options);
+  r.context = {animal: 'Sterrance'};
+
+  try {
+    r.write('foo\n');
+  } finally {
```

TIL :-) I always thought ignoring `catch` is the same as ignoring errors and I have no idea why I used to think that :( Thanks for enlightening :)